tests/e2e/base.py

```diff
@@ -37,6 +46,51 @@
 ]
 
 
+class Parameters:
+    param_d: Dict[str, Any]
+
+    def __init__(self, params: List[ParameterSet]) -> "Parameters":
```

Comment:
The correct type hint for `__init__` return is `None` - this method never returns anything